config/locales/en.yml

```diff
@@ -1082,6 +1082,33 @@ en:
     register_call:
       selling_on_ofn: "Interested in getting on the Open Food Network?"
       register: "Register here"
+    footer:
+      footer_global_headline: "OFN Global"
```

Comment:
You could have removed the `footer_` prefix from all these variables, but that doesn't really matter.

Reply:
ouch, that's right

Reply:
I thought about that BUT I didnt do it because I am not sure transifex will pick up the translations if we change the translation key :-)

Reply:
Transifex should be smart enough.